pkg/prometheus/testdata/RemoteWriteConfig_v2.48.0_1.golden

```diff
@@ -0,0 +1,14 @@
+global:
+  evaluation_interval: 30s
+  scrape_interval: 30s
+  external_labels:
+    prometheus: default/test
+    prometheus_replica: $(POD_NAME)
+scrape_configs: []
+remote_write:
+- url: http://example.com
+  remote_timeout: 30s
+  azuread:
+    managed_identity:
+      client_id: client-id
+    cloud: AzureGovernment
```

Comment:
Is this file being used somewhere? I believe you are using the other one (`RemoteWriteConfigAzureADOAuth_v2.48.0_1.golden`), right?

Reply:
Wops! removed.